docs/api/features/manifold.py

```diff
@@ -146,13 +149,27 @@ def plot_all_manifolds(self, path="images"):
         for algorithm in MANIFOLD_ALGORITHMS:
             self.plot_manifold_embedding(algorithm)
 
+    def manifold_quick_method(image="manifold_quick_method.png"):
```

Comment:
Great work here! But here are a few suggestions.
1. Because we are using `concrete` dataset here, would it be better if we use `dataset_example`(line 47)? Because this class is defined for SCurve examples.
2. If we write different functions this would generate a lot of repeated code. Like the portion for loading dataset. Instead, we can manipulate the `dataset_example`(by adding something like `quick` param) and reduce a lot of repeated code.
Although your code is perfectly working, it is always better to avoid repeated code. Let me know your thoughts about this!